interfaz_controller/interfaz_controller.py

```diff
@@ -1,21 +1,19 @@
 import xlwings as xw
 
 
-def main():
+def main(workbook: xw.Book = None):
     wb = xw.Book.caller()
     sheet = wb.sheets[0]
-    if sheet["A1"].value == "Hello xlwings!":
-        sheet["A1"].value = "Bye xlwings!"
-    else:
-        sheet["A1"].value = "Hello xlwings!"
-
-
-@xw.func
-def hello(name):
-    return f"Hello {name}!"
+    area = float(sheet['B4'].value)
+    h = float(sheet['B5'].value)
+    poro = float(sheet['B6'].value)
+    sw = float(sheet['B7'].value)
+    boi = float(sheet['B8'].value)
+    sheet['B10'].value = 7758 * area * h * poro * (1 - sw) / boi
```

Comment:
I don't see how can I calculate this within the excel file. You need to add a button for it

Reply:
![image](https://user-images.githubusercontent.com/57779200/164580629-f8945516-f8ab-4220-9286-85a97a76f96e.png)

It was added a button

Reply:
Ifyou hit it, it calculates the Stoiip